### PR TITLE
Insurance

### PR DESCRIPTION
**Pull request by hhummel**
Created a new model for insurer, and added field for insurance coverage and insurer to participant.  Fake data, views, serializer, url and tests.

### CODE REVIEW

File: core/tests/insurers.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import Group, User
+from core.tests.base import BaseTestCase
+from core.models.insurer import Insurer
+
+
+class InsurerssTestCase(BaseTestCase):
```
Comment (mattusifer) [inline core/tests/insurers.py]: typo "Insurerss"

File: core/tests/insurers.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        url = reverse('insurer-list')
+        data = {'name': "InsureCo, 'is_active': true}
+        response = self.client.post(url, data, format='json',follow=True, **headers)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(Insurer.objects.count(), 1)
+
+    def test_insurerss_api_when_authed_as_front_desk(self):
+        headers = self.auth_headers_for_user('front_desk')
+        response = self.client.get('/api/insurerss', follow=True, **headers)
```
Comment (mattusifer) [inline core/tests/insurers.py]: another typo (and the test name)

Comment (hhummel) [reply core/tests/insurers.py]: Ugh!

Comment (mattusifer) [reply core/tests/insurers.py]: This guy still has a typo in it (the URL)

**Comment by hhummel** [comment]
Ready

File: core/tests/insurers.py (hunk 2 above)
Comment (hhummel) [reply core/tests/insurers.py]: Yes

Comment (hhummel) [reply core/tests/insurers.py]: Fixed

**Pull request merged**